.github/workflows/helm-template-generate-k8s-manifests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Generate K8S Manifests from Helm Charts
+
+on:
+  # Run this when PR operations are done
+  pull_request:
+    types: [ opened, synchronize, reopened, edited ]
```

Comment:
Is there any chance that this action can cause recursion (e.g. PR gets pushed, an auth-push happens for generated files, causing another GH action... )? Or is the auto-push operation essentially a no-op if there are no changes to the generated manifests?

Reply:
> the auto-push operation essentially a no-op if there are no changes to the generated manifests?
^This